### DISCUSSION TITLE
Suggestion: New Pull Request Diff Review Option: `Hybrid`

### DISCUSSION THREAD

**Discussion by gulsahsarsilmaz** [community/community#4722] [Pull Requests]
Hello 👋  ,

I was curious if it would be possible to consider adding a new third option for Diff Reviewing: **Hybrid Diff View**, to Pull Request Diff View Mode. With the best of my knowledge, the way I had in mind is as follows:

* If the file is new, show Unified Diff View
* If the file is modified (already existed) show the Split Diff View

**The Reason:**

* When we add new files, the content of the file for viewing on the side becomes a bit hard to visualize the changes. It would be really useful to be able to just visualize those files in Unified Diff View Mode to get a better review experience.

What are your thoughts on this?

**Comment by Thiht** [comment]
This would be really useful.
I'd go a step further and simply always display new and deleted files with the unified view. There's no reason to use the split view for new and deleted files since there's nothing to compare.

**Reply by Bobbyjuba** [reply]
I would like to suggest that if there are only additions or only deletions within a file, then display it as Unified - or at least make this an option. In both Unified and Split view, the diff presentation is the same when viewing a file that doesn't have both additions and deletions.
